grafana-datasource/compose.yaml

```diff
@@ -3,7 +3,7 @@ services:
     image: spiceai/spiceai
     container_name: spice
     ports:
-      - "5000:3000"
+      - "5000:5000"
```

Comment:
Why is this 5000? Is that the default Grafana port?

Reply:
It should be 8090 - updating it now.  Its overridden below to 5000 because previously grafana's use of 3000 conflicted with the Spice runtime port.